simulator/mips/mips_instr.h

```diff
@@ -131,10 +131,21 @@ class BaseMIPSInstr
         MIPSRegister get_dst2_num() const { return dst2; }
         std::string_view get_disasm() const { return static_cast<std::string_view>( disasm); }
 
-        /* Checks if instruction can change PC in unusual way. */
-        bool is_jump() const { return operation == OUT_J_JUMP      ||
-                                      operation == OUT_R_JUMP      ||
-                                      operation == OUT_BRANCH;}
+	/* Checks if instruction can change PC in unusual way. */	
+
+	bool is_jump() const { return operation == OUT_J_JUMP ||
```

Comment:
I suggest to use rewrite it as
```c++
return is_direct_jump() || is_direct_branch() || is_indirect_jump();
```
so less code should be updated if `operation` enumeration format changes.